mtbf_job_runner.py

```diff
@@ -87,11 +87,16 @@ def add_7mobile_action(self):
 
     @action(enabled=False)
     def change_memory(self):
-        #TODO: use native adb/fastboot command to change memory?
+        # This function only work in flame
+        # TODO: use native adb/fastboot command to change memory?
         # Make sure it's in fastboot mode, TODO: leverage all fastboot command in one task function
+        memory = 512 # default set 512
+        if 'MEM' in os.environ:
+            memory = os.environ['MEM']
+        elif self.settings['change_memory']['enabled'] and 'memory' in self.settings['change_memory']:
```

Comment:
good point and encourage you to provide question while reviewing.

I suppose the priority is more dynamic > static, for instance

command line argument > file loaded argument > system default

if you're an operation engineer, when you find something wrong you might need to find correct combination of arguments and then modify them in your config files.